lib/ansible/utils/display.py

```diff
@@ -258,6 +259,8 @@ def deprecated(self, msg, version=None, removed=False):
         if not removed:
             if version:
                 new_msg = "[DEPRECATION WARNING]: %s. This feature will be removed in version %s." % (msg, version)
+            elif date:
+                new_msg = "[DEPRECATION WARNING]: %s. This feature will be removed in a release after %s." % (msg, date)
```

Comment:
Should we state that it will be removed in a major release after x like?

```suggestion
                new_msg = "[DEPRECATION WARNING]: %s. This feature will be removed in a major release after %s." % (msg, date)
```

Reply:
That's true for collections. But is 2.11.0 a minor or a major release of Ansible-base?

Reply:
Technically from a semver perspective it's a minor release but Ansible doesn't strictly follow semver as we consider the minor part of 2.x being a "major" release. Others might disagree with me here though. I guess it's a question as to whether we see this particular deprecation being done within Ansible itself or only for collections.

Maybe for now we just leave it as is and revisit the message in the future when we don't have a deadline coming up.

Reply:
With #69680 (adjusted for dates) we would be able to distinguish between ansible-base versions and collection versions. But yeah, let's revisit this somewhen after the freeze :)